lib/fathead/less/script.js

```diff
@@ -12,20 +12,26 @@ const cheerio = require("cheerio");
 })();
 
 function makeOutput(docs) {
+	let logo = `http://lesscss.org/public/img/logo.png`;
 	return docs.map(({article, category, abstract, url}) => {
 		return [
-			`${article}\tA\t\t\t${category}\t\t\t\t\t\thttp://lesscss.org/public/img/logo.png\t${abstract}\t${url}\n`
-		];
+			`${article}\tA\t\t\t${category}\t\t\t\t\t\t${logo}\t${abstract}\t${url}\n`
+		].concat(getRedirects(article)).join("");
 	}).join("");
 }
 
+function getRedirects(article) {
+	return ["css", "lesscss", "less css"]
```

Comment:
Actually we shouldn't use `css` as a trigger as it will overlap with the CSS Fathead which shares some article titles.

Please remove the redirect code.